<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Latitude:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QSpinBox" name="spinBox"/>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox_2"/>
        </item>
        <item row="2" column="5">
         <widget class="QSpinBox" name="spinBox_3"/>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBox_4"/>
        </item>
        <item row="3" column="3">
         <widget class="QSpinBox" name="spinBox_5"/>
        </item>
        <item row="3" column="5">
         <widget class="QSpinBox" name="spinBox_6"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Longitude:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QSpinBox" name="spinBox_7"/>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QSpinBox" name="spinBox_8"/>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="comboBoxBuoyType">
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>D</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Buoy type:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Speed:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Direction:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>knots</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>degrees</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>ยบ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>'</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>''</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>ยบ</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>'</string>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>''</string>
          </property>
         </widget>
        </item>
        <item row="7" column="5" colspan="2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Reset values</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3" colspan="2">
         <widget class="QPushButton" name="pushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Add buoy</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7" rowspan="6">
         <widget class="QListWidget" name="listWidget">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="7">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Delete Buoy</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="6">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Sonobuoy simuform&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2" rowspan="3">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Calibration simuform&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>N/R level</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_17">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Volume level</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QDial" name="di_nrLevel"/>
        </item>
        <item row="2" column="1">
         <widget class="QDial" name="di_volumeLevel"/>
        </item>
        <item row="3" column="0">
         <widget class="QLCDNumber" name="lcdNumber"/>
        </item>
        <item row="3" column="1">
         <widget class="QLCDNumber" name="lcdNumber_2"/>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label_18">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Balance level</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QSlider" name="hs_balanceLevel">
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_19">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Ampli level</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_20">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Signal echo</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QDial" name="di_ampliLevel"/>
        </item>
        <item row="7" column="1">
         <widget class="QDial" name="di_signalEcho"/>
        </item>
        <item row="8" column="0">
         <widget class="QLCDNumber" name="lc_ampliLevel"/>
        </item>
        <item row="8" column="1">
         <widget class="QLCDNumber" name="lc_signalEcho"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_21">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Buffer size:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QSpinBox" name="sb_bufferSize"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_22">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Sample rate:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QSpinBox" name="sb_sampleRate"/>
        </item>
        <item row="11" column="0" colspan="2">
         <widget class="QProgressBar" name="pb_left">
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>Left channel %p%</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="2">
         <widget class="QProgressBar" name="pb_right">
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string>Right channel %p%</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0" colspan="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="14" column="0" colspan="2">
         <widget class="QPushButton" name="reset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0" colspan="2">
         <widget class="QPushButton" name="apply">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="5" column="2" rowspan="4">
         <widget class="QDial" name="dial_5"/>
        </item>
        <item row="5" column="1" rowspan="4">
         <widget class="QDial" name="dial_6"/>
        </item>
        <item row="5" column="0" rowspan="4">
         <widget class="QDial" name="dial_7"/>
        </item>
        <item row="7" column="4">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Limiter</string>
          </property>
         </widget>
        </item>
        <item row="8" column="4">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Phantom</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Mono</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Mon SW</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2" colspan="3">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Save configuration</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Sonar simuform&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" rowspan="2" colspan="3">
         <widget class="QListWidget" name="listWidget_2">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="currentRow">
           <number>0</number>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Configuration alfa</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Configuration beta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Configuration gamma</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Configuration delta</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_26">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Output</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_25">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Volume</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_24">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Sens</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QProgressBar" name="progressBar_5">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QProgressBar" name="progressBar_4">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QProgressBar" name="progressBar_3">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="4" column="3" rowspan="6">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="3">
       <widget class="Line" name="line_2">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>30</number>
        </property>
        <property name="midLineWidth">
         <number>4</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line_5">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>30</number>
        </property>
        <property name="midLineWidth">
         <number>4</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Reset_all">
   <property name="text">
    <string>&amp;Reset all!!!</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
